components/PlaybackTask.brs

```diff
@@ -158,15 +159,42 @@ sub onTruexEvent(event)
         "OPTIN": "optIn",   ' User opts in to choice card
         "OPTOUT": "optOut", ' User opts out of choice card
         "USERCANCELSTREAM": "userCancelStream", ' User exits playback. EG. Typically "back" on choice card
-        "SKIPCARDSHOWN": "skipCardShown"
+        "SKIPCARDSHOWN": "skipCardShown",
+        "VIDEOEVENT": "videoEvent"
     }
-
+    
+    rafEventType = invalid
     if eventType = types.ADFREEPOD
         m.skipAds = true
     else if eventType = types.ADCOMPLETED OR eventType = types.NOADSAVAILABLE OR eventType = types.ADERROR
         playContentStream()
     else if eventType = types.USERCANCELSTREAM
         exitContentStream()
+    else if eventType = types.VIDEOEVENT
```

Comment:
I could be wrong, but I think these should be engagement events instead of video event. 

1. Not all Engagement has 1 and only 1 video
2. Not all most of the videos are not 30s in length

I guess I am saying, if we are using DAR to measure true[X] engagement as if it were a video ad, then we should be using engagement quartile event instead of video's

Reply:
I think video event is more accurate, one cannot usually tell that it is a video ad per se, so using that seems safer, and just as easy for the caller to use.

Reply:
There is a larger discussion that is stemming from this.  What should Truex/Nielsen should be tracking for these vast beacons.  Instead of using "started" for the init case.  It would use "adStarted" from TAR.  I agree with Kyle in that engagements make more sense to track than the contents of the engagement.  And not all engagements have videos, should those get tracked?

There is a bit of mixed information though, so this is something to be clarified.